### ISSUE TITLE
Typescript can't find lib.es6.d.ts during bundle

### ISSUE THREAD

**Issue by mikeyoon** [jspm/jspm-cli#1432]
Since version 0.16.20, I get this error whenever I do a bundle (with plugin-typescript):

TypeScript ENOENT: no such file or directory, open '[project root]/typescript/lib/lib.es6.d.ts'

Looks like it should be looking in node_modules instead of the project root. Not sure if this is a SystemJS or JSPM issue though. Dev workflow with System.import works fine though and catches compilation errors as expected.

**Comment by guybedford** [comment]
Are you using the TypeScript plugin here or the SystemJS TypeScript support?

**Comment by mikeyoon** [comment]
This is with plugin-typescript, not the built-in one with system.js.

**Comment by mikeyoon** [comment]
Looks like a change in SystemJS somewhere requires an update on the plugin-typescript side. https://github.com/frankwallis/plugin-typescript/issues/84. I think this issue can be closed if he's right.

**Comment by frankwallis** [comment]
This is now resolved in plugin-typescript.

@guybedford I don't really know what the issue was here, it seemed as though the map configuration had gone missing when bundling, because ```System.normalize``` was not returning the same results when bundling as it was in the browser. I resolved it by switching plugin-typescript to be published in the system.register format, so possibly it was an issue affecting commonjs only?
